tests/functionality_tests/simple_turn_test.py

```diff
@@ -0,0 +1,134 @@
+            0, w, robot_state.max_velocity, robot_state.radius
+        )
+
+        travel(robot_state, 0, robot_state.time_step * limited_cmd_w)
+        if not robot_state.is_sim:
+            robot_state.motor_controller.spin_motors(limited_cmd_w, 0)
+            time.sleep(10)
+        
+        database.update_data(
+            "state", robot_state.state[0], robot_state.state[1], robot_state.state[2]
+        )
+
+    # re-enable after finishing turning
+    robot_state.enable_obstacle_avoidance = True
+
+
+target = [0, 0]
+add_to_x = False
+gps_noise_range = 0.3
+
+
+class TestMoveToTargetNode(unittest.TestCase):
+    def test1(self):
+        r2d2_state = Robot_State(
+            xpos=32.444250,
+            ypos=-44.483682,
+            heading=0,
+            max_velocity=0.5,
+            radius=0.2
+        )
+        r2d2 = Robot(robot_state=r2d2_state)
+        database = DataBase(r2d2)
+
+        allowed_heading_error = 0.1
+        simpler_turn_to_target_heading(
+            r2d2_state, math.pi, allowed_heading_error, database
+        )
+
+    def test2(self):
+        r2d2_state = Robot_State(
+            xpos=42.444250,
+            ypos=-54.483682,
+            heading=math.pi / 2,
+            max_velocity=0.5,
+            radius=0.2
+        )
+        r2d2 = Robot(robot_state=r2d2_state)
+        database = DataBase(r2d2)
+
+        allowed_heading_error = 0.1
+        simpler_turn_to_target_heading(
+            r2d2_state, math.pi / 2, allowed_heading_error, database
+        )
+
+    def test3(self):
+        r2d2_state = Robot_State(
+            xpos=12.444250,
+            ypos=-64.483682,
+            heading=0,
+            max_velocity=0.5,
+            radius=0.2,
+        )
+        r2d2 = Robot(robot_state=r2d2_state)
+        database = DataBase(r2d2)
+
+        allowed_heading_error = 0.1
+        simpler_turn_to_target_heading(
+            r2d2_state, -math.pi / 2, allowed_heading_error, database
+        )
+    
+    def test4(self):
+        r2d2_state = Robot_State(
+            xpos=-32.444250,
+            ypos=44.483682,
+            heading=math.pi / 2,
+            max_velocity=0.5,
+            radius=0.2,
+        )
+        r2d2 = Robot(robot_state=r2d2_state)
+        database = DataBase(r2d2)
+        allowed_heading_error = 0.1
+        simpler_turn_to_target_heading(
+            r2d2_state, -math.pi, allowed_heading_error, database
+        )
+    
+    def test5(self):
+        r2d2_state = Robot_State(
+            xpos=22.444250,
+            ypos=44.483682,
+            heading=0,
+            max_velocity=0.5,
+            radius=0.2,
+        )
+        r2d2 = Robot(robot_state=r2d2_state)
+        database = DataBase(r2d2)
+        allowed_dist_error = 0.1
+        simpler_turn_to_target_heading(
+            r2d2_state, math.pi / 2, allowed_dist_error, database
+        )
+
+# Takes approx. 9 seconds for 5 tests
+
+if __name__ == "__main__":
+    unittest.main()
```

Comment:
add TODO: make a physical script and document expected behavior. Set a bool to toggle using the physical script vs the unit test